tests/e2e/test_rosacli_cluster.go

```diff
@@ -569,3 +569,77 @@ var _ = Describe("Classic cluster creation negavite testing",
 			Expect(out.String()).To(ContainSubstring("to create compatible roles and try again"))
 		})
 	})
+var _ = Describe("Create cluster",
+	labels.Day1,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			rosaClient         *rosacli.Client
+			clusterService     rosacli.ClusterService
+			machinePoolService rosacli.MachinePoolService
+		)
+		BeforeEach(func() {
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			clusterService = rosaClient.Cluster
+			machinePoolService = rosaClient.MachinePool
+		})
+		It(" User can set availability zones to create rosa single-az cluster - [id:52691]",
+			labels.Critical,
+			labels.NonHCPCluster,
+			func() {
+				clusterName := "ocp-52691"
+				region := "us-west-2"
+				availabilityZones := "us-west-2b"
+				mpID := "mp-52691"
+				machineType := "m5.2xlarge"
+
+				By("Run command to check help message of availability zones")
+				output, err, _ := clusterService.Create("", "-h")
+				Expect(err).ToNot(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring("--availability-zones"))
+
+				By("Create cluster with availability zones")
+				output, err, _ = clusterService.Create(
+					clusterName, "--region", region, "--availability-zones", availabilityZones,
+				)
+				Expect(err).To(BeNil())
+				Expect(output.String()).To(ContainSubstring(fmt.Sprintf("Cluster '%s' has been created.", clusterName)))
+
+				profilehandler.WaitForClusterReady(rosaClient, clusterName, 60)
```

Comment:
Please, use profile. We already abandon this way for cluster creation.

Reply:
fixed. create cluster with profile.